### PR TITLE
Add End to End Test cases

### PR DESCRIPTION
**Pull request by alvitazwar**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          NODE_VERSION: 16
+          RCLONE_CONFIG: ${{ secrets.RCLONE_CONFIG }}
+
+      - name: Archive HTML Report on failure
+        if: failure()
+        uses: actions/upload-artifact@v1
+        with:
+          name: report
+          path: ./tests/e2e-playwright/artifacts
+
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:50]: @alvitazwar 

Remove extra spaces

**Review by pooja-muchandikar — COMMENTED** [review]
@alvitazwar Remove `.DS_Store` and `Artifacts` folder.. 

Please make note not to commit these files in PR..

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (alvitazwar) [reply .github/workflows/e2e.yml]: Removed

**Comment by alvitazwar** [comment]
@pooja-muchandikar I addressed those feedback. If everything looks Fine, then we can merge this PR and close the task

**Review by pooja-muchandikar — APPROVED** [review]

**Pull request merged**